exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverExporter.java

```diff
@@ -72,13 +73,13 @@ public static void createAndRegisterWithCredentials(Credentials credentials, Str
    * Stackdriver exporter can be registered at any point.
    *
    * <p>This uses the default application credentials see {@link
-   * GoogleCredentials#getApplicationDefault()}. If you do not have default application credentials
+   * GoogleCredentials#getApplicationDefault}. If you do not have default application credentials
    * configured use {@link #createAndRegisterWithCredentials(Credentials, String)}.
    *
    * @param projectId the cloud project id.
    * @throws IllegalStateException if a Stackdriver exporter is already registered.
    */
-  public static void createAndRegister(String projectId) throws IOException {
+  public static void createAndRegisterWithProjectId(String projectId) throws IOException {
```

Comment:
Should createAndRegisterWithCredentials also be renamed, for consistency, i.e., createAndRegisterWithCredentialsAndProjectId?

Reply:
Done.